FirebaseRemoteConfig.podspec

```diff
@@ -53,6 +53,7 @@ app update.
   s.dependency 'FirebaseInstallations', '~> 10.0'
   s.dependency 'GoogleUtilities/Environment', '~> 7.8'
   s.dependency 'GoogleUtilities/NSData+zlib', '~> 7.8'
+  s.dependency 'FirebaseRemoteConfigInterop', '~> 10.0'
```

Comment:
This should be 10.20 since the new pod doesn't exist before that. 

The new podspec, with its local path, needs to be added to scripts/localize_podfile.swift and various Podfile's in the repo.